omod/src/main/java/org/openmrs/module/dhisconnector/web/controller/DHISConnectorController.java

```diff
@@ -332,6 +304,28 @@ public void manageMappings(ModelMap model) {
 		model.addAttribute("showLogin", (Context.getAuthenticatedUser() == null) ? true : false);
 	}
 
+	@RequestMapping(value = "/module/dhisconnector/manageMappings", method = RequestMethod.POST)
+	public void manageMappings(ModelMap model,
+							  @RequestParam(value = "mapping", required = false) MultipartFile mapping) throws IOException {
```

Comment:
did this line formatting come in by mistake?

Reply:
I'll reformat it, thank you

Reply:
Resolved with commit 457f8cf